test/data/logfile.asc

```diff
@@ -1,18 +1,18 @@
-date Sam Sep 30 15:06:13.191 2017

```

Comment:
@christiansandberg Do ASC files work with any line endings? It should work with our tools, but do they work with the proprietary tools?

Reply:
I made the change blind, hoping nothing would regress :stuck_out_tongue: clearly I was wrong. This wasn't flagged by the RPM linter, so I can roll back this file.